.gitlab-ci.yml

```diff
@@ -2,38 +2,35 @@ stages:
   - deploy
   - test
 
-variables:
-  SKIP_DRIVER: Remote  # set to SauceLabs to instead only run local selenium
-
-.base-url:
-  only:
-    variables:
-      - $BASE_URL
-
-.default-test-variables:
-  variables:
-    DRIVER: SauceLabs
-    MARK_EXPRESSION: not headless and not download
-    TEST_IMAGE: mozmeao/bedrock_test:881f85ceceac17568056d42ba1151e720d9e3ed1
-    PYTEST_PROCESSES: "4"
-
-.default-test-variables-local:
-  extends: .default-test-variables
-  variables:
-    DRIVER: Remote
-
 .deploy:
+  stage: deploy
   script:
     - kubectl apply -f ${DEPLOYMENT}
     - bin/rollout-status.sh ${DEPLOYMENT}
-  stage: deploy
+    # Used by next steps in test stage.
+    - echo "BASE_URL=${BASE_URL}" > build.env
+  retry: 2
+  artifacts:
+    reports:
+      dotenv: build.env
+    when:
+      on_success
+    expire_in:
+      15 mins
```

Comment:
Why keep an artifact for only 15 min?

Reply:
If this relates to a screenshot or html report, it would be useful to keep those around at least for a day or so.

Reply:
this is different. this is an artifact for when the deployment succeeds.

Reply:
This artifact is used to pass `BASE_URL` to the `test` stage jobs. It's not really useful to keep longer. The output from the tests are kept for a week